workflow/run_simulation.rb

```diff
@@ -125,22 +128,22 @@ def run_workflow(basedir, rundir, hpxml, debug, skip_validation, add_comp_loads,
   end
 
   options[:hourly_outputs] = []
-  opts.on('--hourly NAME', 'Request hourly output category* or EnergyPlus output variable; can be called multiple times') do |t|
```

Comment:
Is this supposed to support a single `--hourly` argument with a combination of output variables and meters? If so, it doesn't appear to be working. For example, `--hourly "Electricity:Facility" --hourly "Zone People Total Heating Energy"` works but `--hourly "Electricity:Facility, Zone People Total Heating Energy"` does not.

Reply:
Do you think it's important to retain this? I'm tempted to disallow it and just require the use of multiple explicit `--hourly` requests.

Reply:
That's fine. Just checking. It looked like, e.g., `--hourly "Zone People Occupant Count, Zone People Total Heating Energy"` worked, and I wasn't sure if that was intentional or not.

Reply:
Disallowed commas in the latest commit. This will prevent possible user confusion if you try to mix output categories with EnergyPlus output variables/meters (e.g., `--hourly "enduses, fuels"` or `--hourly "Zone People Total Heating Energy, enduses"`).